tests/sentry/snuba/test_discover.py

```diff
@@ -1625,131 +1629,144 @@ def test_function_conditions(self, mock_query):
 
     @patch("sentry.snuba.discover.raw_query")
     def test_find_histogram_min_max(self, mock_query):
-        # no rows returned from snuba
-        mock_query.side_effect = [{"meta": [], "data": []}]
-        values = discover.find_histogram_min_max(
-            ["measurements.foo"], None, None, "", {"project_id": [self.project.id]}
-        )
-        assert values == (None, None)
+        for array_column in ARRAY_COLUMNS:
+            # no rows returned from snuba
+            mock_query.side_effect = [{"meta": [], "data": []}]
+            values = discover.find_histogram_min_max(
+                [f"{array_column}.foo"], None, None, "", {"project_id": [self.project.id]}
+            )
+            assert values == (None, None)
```

Comment:
asserts take an optional message, could we use that so we can tell when this fails on measurements vs op breakdowns?
eg.
`assert a % 2 == 0, "value was odd, should be even"`